src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static void getRandomBall() {
+        Random random = new Random();
+        int value = new Random().nextInt(100);
+        System.out.println(ColorSupplier.getRandomColor() + " " + value);
```

Comment:
```suggestion
public class Lottery {
    private static final Random random = new Random();
    private static final ColorSupplier colorSupplier = new ColorSupplier();
    
    public void getRandomBall() {
        int value = random.nextInt(100);
        System.out.println(colorSupplier.getRandomColor() + " " + value);
```
the same as in ColorSupplier